### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by manonthemat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,4 +95,4 @@ It will create a new file called `JMdict.json` in the same directory.
 
 # Download
 
-I have already created one for the English **only** JMdict file. You can grab it from [here](https://droppy.cryf.in/$/qfxsg).
\ No newline at end of file
+I have already created one for the English **only** JMdict file. You can grab it from [here](https://droppy.cryf.in/$/qfxsg).
```
Comment (smachave) [inline README.md]: What is the change in this line? The GitHub diff checker is not showing any changes.

**Review by smachave — CHANGES_REQUESTED** [review]
Hey! Nice to see someone contribute to this repo. Also, cheers to your first contribution!

Had a question regarding your changes :)

File: README.md (hunk 1 above)
Comment (manonthemat) [reply README.md]: That was an unintended change that happened when I saved the file in vim.
Per default, it adds a newline at the end of the file. I'll commit a fix.

Comment (smachave) [reply README.md]: Ah I see. Thank you!

**Review by smachave — APPROVED** [review]

**Pull request merged**